genestack_client/genestack_connection.py

```diff
@@ -88,6 +88,21 @@ def whoami(self):
         """
         return self.application('genestack/signin').invoke('whoami')
 
+    @staticmethod
+    def _handle_inactive_user_auth(auth_result):
+        """
+        Check authentication response and throw descriptive exception for inactive users.
+
+        :param auth_result: authentication response
+        """
+        if auth_result.get('userDeactivated'):
+            raise GenestackAuthenticationException(
+                'User\'s Genestack account was deactivated.'
```

Comment:
Check Bobby example for quotes  https://trac.genestack.com/wiki/DevStuff/CodeStyle/Python

Reply:
Don't add a period at the end of error message.